docs/customize/index.md

```diff
@@ -11,6 +11,7 @@ InvenioRDM instance so that it integrates into their organizational environment.
 - [Authentication](authentication.md) - integrate with your institutional authentication system.
 - [DOI registration](dois.md) - mint DOIs with your institution's prefix.
 - [Application data](application_data.md) - use custom controlled vocabularies, default users and other institution specific data.
```

Comment:
I think we should rename ``Application data`` to ``Vocabularies`` and add subpages:

- Styling
- Authentication
- DOI registration
- Vocabularies
    - Affiliations (ROR)
    - Subjects
    - Resource types
    - ...

Reply:
I am moving the `users` part of `applictaion_data` to: install->configuration
Then proceding to implement this structure:
```
Vocabularies
  Affiliations (ROR)
  Subjects
  Resource types
...
```